.github/workflows/ci.yml

```diff
@@ -134,6 +135,60 @@ jobs:
           name: bin
           path: bin/
 
+  path_filter:
+    name: Filters for fronted specs
+    runs-on: ubuntu-latest
+    outputs:
+      views: ${{ steps.changes.outputs.views }}
+      js: ${{ steps.changes.outputs.js }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - uses: dorny/paths-filter@v2
```

Comment:
Seems like a well regarded action 👍  seeing both Sentry and Google Chrome is using it: https://github.com/dorny/paths-filter ([though Sentry forked it](https://github.com/getsentry/paths-filter)) :)

Reply:
Yeah, that's what I noticed and why I allowed myself to use it :)